OpenRA.Mods.Common/LoadScreens/LogoStripeLoadScreen.cs

```diff
@@ -25,6 +25,7 @@ public sealed class LogoStripeLoadScreen : BlankLoadScreen
 		float2 logoPos;
 		Sheet sheet;
 		Sprite stripe, logo;
+		string defaultMessage = "Loading...";
```

Comment:
Not really. This is a hardcoded fallback string. Nobody is going to add more things to it.